.travis.yml

```diff
@@ -11,9 +11,8 @@ jdk:
   - oraclejdk8
 
 install:
-  - yes | sdkmanager "ndk-bundle"
-  - yes | sdkmanager "cmake;3.6.4111459"
-  - yes | sdkmanager "lldb;3.1"
+  - echo y | sdkmanager "ndk;20.0.5594570"
+  - echo y | sdkmanager "cmake;3.10.2.4988404"
```

Comment:
Can you please explain these changes?

Reply:
CI was failing on my travis with this error:
```
Failed to find package lldb;3.1
```
I can remove the commit if that's OK to you

Reply:
No worries, it's ok, I was just wondering.